### DISCUSSION TITLE
Adding G-Helper to Windows 11 context menu

### DISCUSSION THREAD

**Discussion by eiirk** [seerge/g-helper#2561] [General]
I noticed that my context menu in windows 11 still had the option to "Open Armoury Crate" even though I uninstalled it. Figured you could remove it from the context menu through registry editor. Then I had the thought to add G-Helper and found this guide: 

https://www.windowscentral.com/software-apps/windows-11/how-to-integrate-custom-context-menu-shortcuts-on-windows-11

I did manage to add something, but it doesn't open G-Helper in the same way as when you click it on the icon tray. Seems to only restart the application. I directed the command so it opens the .exe file in a folder on my desktop. 

Does anyone know how to sort out this issue?

**Comment by seerge** [comment]
@eiirk hello,

I don't think you will be able to add that to the context menu. As you have already discovered, it only allows an option to launch a new executable.

While G-Helper is already running, by starting it over - it will only stop existing app, and run a new one. Which is not what you want.

To make G-Helper [main window show from system tray](https://github.com/seerge/g-helper?tab=readme-ov-file#%EF%B8%8F-keybindings) you need to emit ``Ctrl + Shift + F12`` somehow. I don't know if that is possible from context menu :)
